frontend/pages/UserSettingsPage/UserSettingsPage.jsx

```diff
@@ -402,6 +402,15 @@ export class UserSettingsPage extends Component {
           <span
             className={`${baseClass}__version`}
           >{`Fleet ${version.version} • Go ${version.go_version}`}</span>
+          <span className={`${baseClass}__privacy-policy`}>
+            <a
+              href="https://fleetdm.com/legal/privacy"
```

Comment:
ty for using the special link instead of where it redirects to!  This is perfect, since it allows us to change where that redirects to as needed 🙇